Examples/UIExplorer/js/CameraRollView.js

```diff
@@ -94,6 +94,7 @@ var CameraRollView = React.createClass({
         var imageStyle = [styles.image, {width: imageSize, height: imageSize}];
         return (
           <Image
+            key={asset}
```

Comment:
What's the type of `asset`? Is it `Image`? Is it OK to use an `Image` object as a React key? Probably best to use its path or something else?